spec/index.html

```diff
@@ -4582,7 +4582,7 @@ <h4>Descriptions of Resources</h4>
             </div>
           </div>
           <p>which includes the blank node closure for the 
-            <a data-cite="vcard-rdf#" class="inform">vCard vocabulary</a> <code>vcard:N</code>.
+            vCard vocabulary <code>vcard:N</code>.
```

Comment:
Please can be have a link to the vCard vocabulary here.

Reply:
OK, so why not link also to FOAF?